http.go

```diff
@@ -161,3 +155,115 @@ func startHTTPHandler(state *State) {
 	}
 	log.Fatal(r.Run(*httpAddr))
 }
+
+// Round a duration, see original source here: https://play.golang.org/p/WjfKwhhjL5
+func Round(d, r time.Duration) time.Duration {
```

Comment:
Does the [time().Round()](https://golang.org/pkg/time/#Time.Round) function not do what you want here?

Reply:
I don't think so. I was originally trying to do it with time().Round(), but was unable to get the fractional output I was aiming for

Reply:
Take it back it does seem to work